src/library.js

```diff
@@ -402,7 +402,7 @@ addToLibrary({
   // so we cannot override parts of it, and therefore cannot use libc_optz.
 #if (SHRINK_LEVEL < 2 || LINKABLE || process.env.EMCC_FORCE_STDLIBS) && !STANDALONE_WASM && !BULK_MEMORY
 
-#if MIN_CHROME_VERSION < 45 || MIN_EDGE_VERSION < 14 || MIN_FIREFOX_VERSION < 34 || MIN_IE_VERSION != TARGET_NOT_SUPPORTED || MIN_SAFARI_VERSION < 100101
+#if caniuse('js.TypedArray.copyWithin')
```

Comment:
Should all the comments below about version testing be removed, if we are going to trust the caniuse data source?

Reply:
Yeah, I removed such comments in other places but probably missed here.